hacking/target-mm32.md

```diff
@@ -0,0 +1,102 @@
+# MindMotion MM32 Targets
+
+This document
+
+- introduces MM32 processors
```

Comment:
Given the use of bullet points here, please capitalise the first letter of each. For the third point, we would move the "and" to the end of the second.

Please also capitalise: "Black Magic Probe", "Flash", "RAM", and "ID" suitably through out the document.